modules/biobambam/bamsormadup/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: biobambam_bamsormadup
+description: Parallel sorting and duplicate marking
+keywords:
+  - markduplicates
+  - sort
+  - bam
+  - cram
+tools:
+  - biobambam:
+      description: |
+        biobambam is a set of tools for early stage alignment file processing.
+      homepage: https://gitlab.com/german.tischler/biobambam2
+      documentation: https://gitlab.com/german.tischler/biobambam2/-/blob/master/README.md
+      doi: 10.1186/1751-0473-9-13
+      licence: ["GPL v3"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bams:
+      type: files
```

Comment:
```suggestion
      type: file
```

Reply:
I added `files` because some other tools did so to designate a list of possibly multiple inputs. Changed it!